### DISCUSSION TITLE
Questions (wiring, versions, ...)

### DISCUSSION THREAD

**Discussion by zulufoxtrot** [dingo35/SmartEVSE-3.5#152] [General]
Hi!
I just got a SmartEVSE. I wired it like this:

![image](https://github.com/user-attachments/assets/52420538-b751-4aef-9221-a6841b3e0d35)

The contactor is a Schneider R9PCTS20.

The car is plugged through a schuko socket and the factory charge cable. The EVSE is configured in socket mode.

I have a few questions.

1. I see that this repo is a fork of https://github.com/serkri/SmartEVSE-3 which is itself a fork of the original https://github.com/SmartEVSE/SmartEVSE-3.

I don't really understand which one I should use. I want to provide mains metering values through the API. The Serkri fork offers it but it's unclear if this fork does it too. I did not find the API option in the MAINSMET menu of this fork.

2. I can't get the EVSE to close the contactor. I can see that the EVSE provides power on the C1 / C2 pins when I test it with a t-pole tester. I'm not sure of the output voltage of the EVSE. The contactor requires 230V on its A1/A2 pins. The contactor does close if I power it with 230V.

3. Maybe related to point 2: in normal mode, when I press the right button for 2 seconds, the EVSE displays "Press 2 sec to start" then after 2 seconds it just displays "press 2 sec" briefly, then it goes back to "Ready to charge". It looks like nothing is happening. I did manage to get the "charging" view on the Serkri fork, in Solar mode, when providing meters data through the API.

4. There is something I don't understand: how can the EVSE control the power output, if the contactor is a simple on/off relay? I bought the EVSE to make the best use of available solar power. Am I missing something?

Thanks for your help. I don't know much about electrical engineering.

**Comment by dingo35** [comment]
1. This fork offers everything that serkri has and more; flash newest version v3.7.2 and look carefully, API is definitely in the MainsMeter Menu; make sure you are in Smart mode to see all Smart options, and in Solar mode to see all Solar options. Look at the manual links at the bottom of the github.com/dingo35/SmartEVSE-3.5 page.
2. The SmartEVSae puts 230V between C1 and Neutral; you dont need C2 if you only have  one contactor.
3. You are right, I think if you fixed 2 this will work too
4. The CP and PP lines are used to tell the car how much current is available. The car decides how much of that available current it is actually going to use; so you are right, the SmartEVSE tells what is available, but does not know how much is actually consumed. You need an EVMeter for that.

**Reply by zulufoxtrot** [reply]
Thanks for your response.

Point 4 makes a lot of sense now. I thought the EVSE would somehow physically limit available current (but it didn't make sense with my wiring).

One last thing I am confused about: is the EVSE capable of providing power at all *without* the CP/PP lines? Basically just as a relay controller, like on my schematic?

If not, it means the EVSE was designed to work with a Type 2 connector *only*, it cannot work with schuko?

**Comment by dingo35** [comment]
https://en.m.wikipedia.org/wiki/Type_2_connector

**Comment by dingo35** [comment]
You are right, you only need an EVSE controller to operate the CP and PP lines ; schuko doesnt have that, so you wouldnt need a SmartEVSE controller

**Reply by zulufoxtrot** [reply]
Thank you. So I understand that the controller needs CP/PP (and therefore type 2) to operate.
I'll get to work then.
